### ISSUE TITLE
General research on open fonts (both text and SMUFL) to support

### ISSUE THREAD

**Issue by mike-spa** [musescore/MuseScore#16189]
**Task description**

Doing some general research on open fonts (both **text** and **SMUFL** fonts), and consider whether or not we may decide to support them by packaging them with the program.

Examples of text fonts:
- Nepomuk https://github.com/benwiggy/nepomuk
- _(feel free to add)_

Examples of SMUFL fonts:
- Finale Engraver https://makemusic.zendesk.com/hc/en-us/articles/1500013053461-MakeMusic-Fonts-and-Licensing-Information
- _(feel free to add)_

**Comment by Jojo-Schmitz** [comment]
Beside Broadway and Maestro, there are more Finale fonts we might include, check https://usermanuals.finalemusic.com/FinaleMac/Content/Finale/ht-fonts.htm

**Comment by Tantacrul** [comment]
> Besides Broadway, Maestro and Engraver, there are more Finale fonts we might include, also text fonts

Yeah. I'm not very knowledgeable about those. I'd say we want to balance the quantity to make sure we only support the nicer fonts, although I haven't seriously assessed any of the others (not has @oktophonie AFAIK)

Of course we should also build the ability for users to load in whatever they want. I'm only speaking about the ones we package natively (which the vast majority of users will see).

**Comment by Jojo-Schmitz** [comment]
Maybe check https://github.com/Jojo-Schmitz/MuseScore/pull/75, which I stole from  https://github.com/cbjeukendrup/MuseScore/tree/custom-music-fonts

**Comment by Jojo-Schmitz** [comment]
Also Sebastian, requested in https://musescore.org/en/node/371485
